loadingScript.sql

```diff
@@ -0,0 +1,354 @@
+ (1,'Callen','Parrish','callenp@bbd.co.za')
+,(3,'Carlton','Carey','carreyc@bbd.co.za')
+,(1,'Randall','Drew','randalld@bbd.co.za')
+,(1,'Huda','Weiss','hudaw@bbd.co.za')
+,(2,'Graham','Rose','grahamr@bbd.co.za')
+,(2,'Ridwan','Carver','ridwanc@bbd.co.za')
+,(3,'Juliet','Childs','julietc@bbd.co.za')
+,(3,'Susan','Haworth','susanh@bbd.co.za')
+,(1,'Eliana','Cohen','elianac@bbd.co.za')
+,(2,'Priya','Naaidoo','priyan@bbd.co.za')
+,(2,'Khloe','Mackenzie','khloem@bbd.co.za')
+,(1,'Rajesh','Naik','rajeshn@bbd.co.za')
+,(1,'Hamaad','Peel','hamaadp@bbd.co.za')
+,(2,'Kaleem','Noble','kaleemn@bbd.co.za')
+,(3,'Eitan','Hoffman','eitanh@bbd.co.za')
+,(3,'Patrick','Johnson','patrick@bbd.co.za')
+,(1,'Fatima','Noor','fatiman@bbd.co.za')
+,(2,'Eoghan','Rooney','eoghanr@bbd.co.za')
+,(3,'Dakota','Robertson','dakotar@bbd.co.za')
+,(4,'Nurayn','Khan','nuraynk@bbd.co.za')
+,(4,'Pravin','Patel','pravinp@bbd.co.za')
+,(4,'Shilpa','Singh','shilpas@bbd.co.za')
+,(4,'Devi','Govinder','devig@bbd.co.za')
+,(4,'Uzair','Das','uzaird@bbd.co.za')
+,(4,'Mohammed','Ismail','mohmmedi@bbd.co.za')
+,(4,'Gita','Kumari','gitak@bbd.co.za')
+,(4,'Pranay','Giga','pranayg@bbd.co.za')
+,(3,'Jolene','Gurthrie','joleneg@bbd.co.za')
+,(3,'Donald','Pham','donaldp@bbd.co.za');
+
+
+
+INSERT INTO Vendor(OfficeID,AddressID,MenuID,Name,PhoneNumber,WebsiteAddress) VALUES
+ (2,5,1,'Mcdonalds','+27834742989','mcd.co.za')
+,(1,6,1,'Mcdonalds','+27982981829','mcd.co.za')
+,(1,7,2,'King Pie','+27927872980','kingpie.com')
+,(2,8,3,'Nandos','+27837728788','nandos.co.za')
+,(1,9,3,'Nandos','+27029190190','nandos.co.za')
+,(1,10,3,'Nandos','+27298282989','nandos.co.za')
+,(2,11,4,'Spur Steak Ranches','+27990290230','spur.co.za')
+,(1,12,5,'Rocomamas','+27829829289','rocomamas.co.za')
+,(1,13,6,'Romans','+27928928299','romans.com')
+,(2,14,7,'KFC','+27983973999','kfc.co.za')
+,(2,15,8,'Debonairs','+27938293289','debonairs.co.za')
+,(3,16,9,'Wimpy','+447702988329','wimpy.co.uk')
+,(3,17,10,'Wendy''s','+447873837483','wendys.co.uk')
+,(3,18,11,'Papa Johns','+448938298928','papajohns.co.uk')
+,(3,19,12,'Greggs','+440929032988','greggs.com')
+,(3,20,13,'Krispy Kreme','+449298293829','krispykreme.co.uk')
+,(3,21,14,'SpudULike','+448982823727','spudulike.co.uk')
+,(3,22,15,'Yo! Sushi','+448278732891','yosushi.com')
+,(3,23,16,'Wrapchic','+442917071381','wrapchic.com')
+,(3,24,17,'Mcdonalds','+443099340939','mcd.cu.uk')
+,(3,25,18,'Bagel Nash','+442083298329','bagelnash.co.uk')
+,(4,26,19,'KFC','+919039203290','kfc.co.ind')
+,(4,27,20,'The Burger Barn Cafe','+919028080207','burgerbarn.co.ind')
+,(4,28,21,'Terrasinne','+918398239839','terrasinne.co.ind')
+,(4,29,22,'Hotel Sanskruti Misal','+917947384115','sanskruti.com')
+,(4,30,23,'Shree Krishna Pure Veg','+917947241889','krishnaveg.co.ind')
+,(4,31,24,'CHinjabi','+918328211982','chinjabi.com')
+,(4,32,25,'Burger king','+919981891780','burgerking.co.ind');
+
+
+
+INSERT INTO Menu(Title,Summary,CreatedAt,UpdatedAt) VALUES
```

Comment:
Menu's fields changed. We don't have CreatedAt anymore because we decided it is not valuable information, only the most recent update is necessary.